test/box-luatest/gh_6305_autocomplete_metamethod_test.lua

```diff
@@ -0,0 +1,54 @@
+local t = require('luatest')
+local g = t.group()
+local console = require('console')
+
+local function tabcomplete(s)
+        return console.completion_handler(s, 0, #s)
```

Comment:
We use four spaces for indentation in Lua source files. Please fix.

Reply:
fixed